File: scratch_check_blog.py
import os
import re

blog_dir = "blog"
files = os.listdir(blog_dir)

print(f"Checking {len(files)} files in '{blog_dir}'...")

line_num_pattern = re.compile(r'\b\d+:\s')

for file in sorted(files):
    if not file.endswith('.html'):
        continue
    filepath = os.path.join(blog_dir, file)
    with open(filepath, 'r', encoding='utf-8', errors='ignore') as f:
        content = f.read()
    
    # Calculate word count (strip HTML tags first)
    text = re.sub(r'<[^>]+>', ' ', content)
    words = text.split()
    word_count = len(words)
    
    # Find line number patterns
    matches = line_num_pattern.findall(content)
    
    # Print status
    print(f"{file:<80} | Words: {word_count:<6} | Line-num occurrences: {len(matches)}")

File: scratch_check_blog.js
const fs = require('fs');
const path = require('path');

const blogDir = 'blog';
const files = fs.readdirSync(blogDir);

console.log(`Checking ${files.length} files in '${blogDir}'...`);

const lineNumPattern = /\b\d+:\s/g;

files.sort().forEach(file => {
    if (!file.endsWith('.html')) return;
    const filepath = path.join(blogDir, file);
    const content = fs.readFileSync(filepath, 'utf8');
    
    // Calculate word count (strip HTML tags first)
    const text = content.replace(/<[^>]+>/g, ' ');
    const words = text.trim().split(/\s+/).filter(Boolean);
    const wordCount = words.length;
    
    // Find line number patterns
    const matches = content.match(lineNumPattern) || [];
    
    // Print status
    console.log(`${file.padEnd(80)} | Words: ${String(wordCount).padEnd(6)} | Line-num occurrences: ${matches.length}`);
});
